src/server/controllers/RedirectController.ts

```diff
@@ -60,7 +60,7 @@ export class RedirectController implements RedirectControllerInterface {
     req: Express.Request,
     res: Express.Response,
   ) => Promise<void> = async (req, res) => {
-    const { shortUrl }: { shortUrl: string } = req.params
```

Comment:
back then we added this because typescript would by default interpret `shortUrl` as `any` type, which results in zero type-checking

Reply:
`req.params` is now a ParamsDictionary, defined as follows:
```typescript
export interface ParamsDictionary { [key: string]: string; }
```